### ISSUE TITLE
Feature request :  Add an image placeholder component

### ISSUE THREAD

**Issue by AAS97** [SawyerHood/draw-a-ui#25]
Would be amazing to have a dedicated image placeholder. Later could even leverage image generation with DALL-E to create an image from a simple prompt in the component.

**Comment by SawyerHood** [comment]
Yeah this is a great idea! It would be nice if we could just have a route that can generate the images based on urls.
